src/main/scala/cromwell/webservice/WorkflowJsonSupport.scala

```diff
@@ -33,7 +33,7 @@ object WorkflowJsonSupport extends DefaultJsonProtocol {
     }
   }
   implicit val executionEventProtocol = jsonFormat3(ExecutionEventEntry)
-  implicit val callMetadataProtocol = jsonFormat15(CallMetadata)
+  implicit val callMetadataProtocol = jsonFormat16(CallMetadata)
```

Comment:
General comment, not asking @Horneth to do anything here - but since we don't have a whole lot of more room in CallMetadata that's a sign that stuff needs to be refactored